docs/topics/whatsnew21.md

```diff
@@ -10,10 +23,1562 @@ All you need to do is [change the Kotlin version](configure-build-for-eap.md) to
+with classloader or process isolation to ensure safe interaction with the compiler.
+
+#### Using the Gradle Workers API
+
+This example demonstrates how to safely use the Kotlin compiler in a project producing a Gradle plugin.
+First, add a compile-only dependency in your build script.
+This makes the symbol available at compile time only:
+
+```kotlin
+// build.gradle.kts
+dependencies {
+    compileOnly("org.jetbrains.kotlin:kotlin-compiler-embeddable:%kotlinVersion%")
+}
+```
+
+Next, define a Gradle work action to print the Kotlin compiler version:
+
+```kotlin
+import org.gradle.workers.WorkAction
+import org.gradle.workers.WorkParameters
+import org.jetbrains.kotlin.config.KotlinCompilerVersion
+abstract class ActionUsingKotlinCompiler : WorkAction<WorkParameters.None> {
+    override fun execute() {
+        println("Kotlin compiler version: ${KotlinCompilerVersion.getVersion()}")
+    }
+}
+```
+
+Now create a task that submits this action to the worker executor using classloader isolation:
+
+```kotlin
+import org.gradle.api.DefaultTask
+import org.gradle.api.file.ConfigurableFileCollection
+import org.gradle.api.tasks.Classpath
+import org.gradle.api.tasks.TaskAction
+import org.gradle.workers.WorkerExecutor
+import javax.inject.Inject
+abstract class TaskUsingKotlinCompiler: DefaultTask() {
+    @get:Inject
+    abstract val executor: WorkerExecutor
+
+    @get:Classpath
+    abstract val kotlinCompiler: ConfigurableFileCollection
+
+    @TaskAction
+    fun compile() {
+        val workQueue = executor.classLoaderIsolation {
+            classpath.from(kotlinCompiler)
+        }
+        workQueue.submit(ActionUsingKotlinCompiler::class.java) {}
+    }
+}
+```
+
+Finally, configure the Kotlin compiler classpath in your Gradle plugin:
+
+```kotlin
+import org.gradle.api.Plugin
+import org.gradle.api.Project
+abstract class MyPlugin: Plugin<Project> {
+    override fun apply(target: Project) {
+        val myDependencyScope = target.configurations.create("myDependencyScope")
+        target.dependencies.add(myDependencyScope.name, "$KOTLIN_COMPILER_EMBEDDABLE:$KOTLIN_COMPILER_VERSION")
+        val myResolvableConfiguration = target.configurations.create("myResolvable") {
+            extendsFrom(myDependencyScope)
+        }
+        target.tasks.register("myTask", TaskUsingKotlinCompiler::class.java) {
+            kotlinCompiler.from(myResolvableConfiguration)
+        }
+    }
+
+    companion object {
+        const val KOTLIN_COMPILER_EMBEDDABLE = "org.jetbrains.kotlin:kotlin-compiler-embeddable"
+        const val KOTLIN_COMPILER_VERSION = "%kotlinVersion%"
+    }
+}
+```
+
+## Compose compiler updates
+
+### Support for multiple stability configuration files
+
+The Compose compiler can interpret multiple stability configuration files,
+but the `stabilityConfigurationFile` option of the Compose Compiler Gradle plugin previously allowed
```

Comment:
```suggestion
but the `stabilityConfigurationFile` option of the Compose Compiler Gradle plugin was previously allowed
```

Reply:
We're missing a verb here. Does "was" work?

Reply:
Not sure, that it is necessary here.